Neutaro/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Neutaro",
+  "assets": [
+    {
+      "description": "The native token of Neutaro",
+      "denom_units": [
+        {
+          "denom": "uneutaro",
+          "exponent": 0
+        },
+        {
+          "denom": "neutaro",
+          "exponent": 6
+        }
+      ],
+      "base": "uneutaro",
+      "name": "neutaro",

```

Comment:
This can capitalize the 'n' since it will be be displayed as this name on interfaces.

Reply:
also done